packages/utils/functions/token/refreshAccessToken/index.ts

```diff
@@ -0,0 +1,38 @@
+import _axios from 'axios'
+import Cookies from 'js-cookie'
+
+import { IJWTResponse } from '../../../types/jwt'
+
+const REFRESH_TOKEN_URL = '/auth/refresh'
+
+// We create an isolated axios instance to skip the interceptors
+const simpleAxios = _axios.create({
+  baseURL: process.env.NEXT_PUBLIC_API_BASE_URL,
+})
+
+export const refreshAccessToken = async (cookieName: string, refreshCookieName: string) => {
+  const refreshToken = Cookies.get(refreshCookieName)
```

Comment:
will that work server side? maybe we can use the `getToken` instead?

Reply:
I was thinking that this function would be only meant to be used in the client side @anicioalexandre, because you can't use axios in the server side either and need to use nextjs's fetch. We are experimenting with a server-side version of it in a FAVIO project right now: https://bitbucket.org/silverlogic/hcmu-web/pull-requests/38#Lservices/fetch.tsT6, but we want to test it out for a couple of weeks before adding it to baseapp

Reply:
true! That fetch implementation is definitely something we should bring to the BA, let's test it then!